content/en/docs/concepts/cluster-administration/cloud-providers.md

```diff
@@ -99,8 +99,8 @@ Different settings can be applied to a load balancer service in AWS using _annot
 * `service.beta.kubernetes.io/aws-load-balancer-connection-draining-timeout`: Used on the service to specify a connection draining timeout.
 * `service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout`: Used on the service to specify the idle connection timeout.
 * `service.beta.kubernetes.io/aws-load-balancer-cross-zone-load-balancing-enabled`: Used on the service to enable or disable cross-zone load balancing.
-* `service.beta.kubernetes.io/aws-load-balancer-security-groups`: Used to specify the security groups to be added to ELB created. This replaces all other security groups previously assigned to the ELB. Security groups defined here should not be shared between services.
-* `service.beta.kubernetes.io/aws-load-balancer-extra-security-groups`: Used on the service to specify additional security groups to be added to ELB created
+* `service.beta.kubernetes.io/aws-load-balancer-security-groups: Used to specify the security groups to be added to ELB created. This replaces all other security groups previously assigned to the ELB. Security groups defined here should not be shared between services. This annotation also overrides the creation of a uniquely generated security group for this ELB. Only use this annotation if you are providing a unique list of security groups, as the first security group ID on this list will be used as source in the security group rules to permit traffic to target worker nodes (service traffic and health checks). If a non-unique Security Group ID is used for multiple ELBs, only a single permit line will be added to worker node security groups, meaning if you delete any of those ELBs it will delete the single PERMIT line and remove access for all ELBs sharing the same security group ID. This can cause a cross-service outage if not used properly.
```

Comment:
```suggestion
* `service.beta.kubernetes.io/aws-load-balancer-security-groups`: Used to specify the security groups to be added to the ELB created. This replaces all other security groups previously assigned to the ELB. Security groups defined here should not be shared between services. This annotation also overrides the creation of a uniquely generated security group for this ELB. Only use this annotation when providing a unique list of security groups, as the first security group ID on this list is used as a source in the security group rules to permit traffic to target worker nodes (service traffic and health checks). If a non-unique Security Group ID is used for multiple ELBs, only a single permit line is added to worker node security groups. If you delete any of those ELBs it deletes the single PERMIT line and removes access for all ELBs sharing the same security group ID. This can cause a cross-service outage if not used properly.
```